src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRentalCost = 40;
+  const totalCost = days * dailyRentalCost;
```

Comment:
It's better to use `let` instead of `const` for `totalCost` since its value might be changed due to discounts.

Reply:
I wanted to do it, but I received an error "'totalCost' is never reassigned. Use 'const' instead.".